### PR TITLE
Add footer and other stuff

### PR DESCRIPTION
**Pull request by walliski**
Adds a footer to the page.
And weather map.
Has minor problems with footer.

### CODE REVIEW

**Comment by walliski** [comment]
Might cause problems with size of some content... 
And something to actually put in the footer will come later...

**Comment by walliski** [comment]
On screens with low height on screen might have problems with content being hidden behind the footer. Needs to be fixed still... :(

File: apps/rotatelogos/templates/rotatelogos/index.html [hunk 1]
```diff
@@ -0,0 +1 @@
+<img src="{{img_url}}"></img>
```
Comment (oehnstro) [inline apps/rotatelogos/templates/rotatelogos/index.html]: `` has no end tag, but mandatory alt property. Consider changing it to `{{img_name}}`

Also, you make git happier by adding a newline at the end of a file :)

File: manager/templates/infoskarm/index.html [hunk 2]
```diff
@@ -43,4 +57,6 @@
 	<script src="{% static 'infoskarm/clock.js' %}"></script>
 	<script src="{% static 'infoskarm/pages.js' %}"></script>
 	<script src="{% static 'infoskarm/small-screen.js' %}"></script>
+	<script src="{% static 'infoskarm/logos.js' %}"></script>
```
Comment (oehnstro) [inline manager/templates/infoskarm/index.html:60]: JS files are not in repository?

Comment (walliski) [inline manager/templates/infoskarm/index.html:60]: .gitignored...

**Pull request merged**